ArgumentParser : Error in "arg_parse/tool_doc.h" file

By using the ArgumentParser of seqan 2.0.0, I have got this kind of compilation errors : 
```text
/usr/local/include/seqan/arg_parse/tool_doc.h: In member function ‘void seqan::TextToolDocPrinter_::_printSection(std::ostream&, const seqan::ToolDocSection_&)’:
/usr/local/include/seqan/arg_parse/tool_doc.h:362:80: error: no matching function for call to ‘transform(seqan::IteratorDefaultImp_<const seqan::String<char, seqan::Alloc<> >, const seqan::Tag<seqan::Rooted_> >::Type, seqan::IteratorDefaultImp_<const seqan::String<char, seqan::Alloc<> >, const seqan::Tag<seqan::Rooted_> >::Type, std::ostream_iterator<char>&, <unresolved overloaded function type>)’
         std::transform(begin(section._title), end(section._title), out, toupper);
```
It seems that g++ does not recognized `toupper` in this case.
To solve this, it only need to change `toupper` by `::toupper` at lines 362, 1293 and 1322 and change `tolower` by `::tolower` at line 1295 in the file "arg_parse/tool_doc.h" as it is done in the example given there http://en.cppreference.com/w/cpp/algorithm/transform

--- a/include/seqan/arg_parse/tool_doc.h
+++ b/include/seqan/arg_parse/tool_doc.h
@@ -359,7 +359,7 @@ public:
     {
         std::ostream_iterator<char> out(stream);
         stream << '\n' << _toText("\\fB");
-        std::transform(begin(section._title), end(section._title), out, std::toupper);
+        std::transform(begin(section._title), end(section._title), out, static_cast < int(*)(int) > (toupper));
         stream << _toText("\\fP") << '\n';
     }
 
@@ -992,7 +992,7 @@ inline CharString const & getManTitle(ToolDoc & doc)
 }
 
 // --------------------------------------------------------------------------
-// Function addSection()                                              ToolDoc
+// Function addSection()ToolDoc
 // --------------------------------------------------------------------------
 
 /*!
@@ -1290,9 +1290,9 @@ void ManToolDocPrinter_::print(std::ostream & stream, ToolDoc const & doc)
 
     // Print .TH line.
     stream << ".TH ";
-    std::transform(begin(doc._name), end(doc._name), out, std::toupper);
+    std::transform(begin(doc._name), end(doc._name), out, static_cast < int(*)(int) > (toupper));
     stream << " " << doc._manSection << " \"" << doc._date << "\" \"";
-    std::transform(begin(doc._name), end(doc._name), out, std::tolower);
+    std::transform(begin(doc._name), end(doc._name), out, static_cast < int(*)(int) > (tolower));
     stream << " " << doc._version << "\" \"" << doc._manTitle << "\"\n";
 
     // Print NAME section.
@@ -1319,7 +1319,7 @@ void ManToolDocPrinter_::print(std::ostream & stream, ToolDoc const & doc)
         {
             ToolDocSection_ const * sec = static_cast<ToolDocSection_ const *>(entry);
             stream << ".SH ";
-            std::transform(begin(sec->_title), end(sec->_title), out, std::toupper);
+            std::transform(begin(sec->_title), end(sec->_title), out, static_cast < int(*)(int) > (toupper));
             stream << "\n";
             isFirstInSection = true;
         }

--- a/include/seqan/arg_parse/tool_doc.h
+++ b/include/seqan/arg_parse/tool_doc.h
@@ -359,7 +359,7 @@ public:
     {
         std::ostream_iterator<char> out(stream);
         stream << '\n' << _toText("\\fB");
-        std::transform(begin(section._title), end(section._title), out, toupper);
+        std::transform(begin(section._title), end(section._title), out, ::toupper);
         stream << _toText("\\fP") << '\n';
     }
 
@@ -1290,9 +1290,9 @@ void ManToolDocPrinter_::print(std::ostream & stream, ToolDoc const & doc)
 
     // Print .TH line.
     stream << ".TH ";
-    std::transform(begin(doc._name), end(doc._name), out, toupper);
+    std::transform(begin(doc._name), end(doc._name), out, ::toupper);
     stream << " " << doc._manSection << " \"" << doc._date << "\" \"";
-    std::transform(begin(doc._name), end(doc._name), out, tolower);
+    std::transform(begin(doc._name), end(doc._name), out, ::tolower);
     stream << " " << doc._version << "\" \"" << doc._manTitle << "\"\n";
 
     // Print NAME section.
@@ -1319,7 +1319,7 @@ void ManToolDocPrinter_::print(std::ostream & stream, ToolDoc const & doc)
         {
             ToolDocSection_ const * sec = static_cast<ToolDocSection_ const *>(entry);
             stream << ".SH ";
-            std::transform(begin(sec->_title), end(sec->_title), out, toupper);
+            std::transform(begin(sec->_title), end(sec->_title), out, ::toupper);
             stream << "\n";
             isFirstInSection = true;
         }

--- a/include/seqan/arg_parse/tool_doc.h
+++ b/include/seqan/arg_parse/tool_doc.h
@@ -359,7 +359,7 @@ public:
     {
         std::ostream_iterator<char> out(stream);
         stream << '\n' << _toText("\\fB");
-        std::transform(begin(section._title), end(section._title), out, ::toupper);
+        std::transform(begin(section._title), end(section._title), out, std::toupper);
         stream << _toText("\\fP") << '\n';
     }
 
@@ -1290,9 +1290,9 @@ void ManToolDocPrinter_::print(std::ostream & stream, ToolDoc const & doc)
 
     // Print .TH line.
     stream << ".TH ";
-    std::transform(begin(doc._name), end(doc._name), out, ::toupper);
+    std::transform(begin(doc._name), end(doc._name), out, std::toupper);
     stream << " " << doc._manSection << " \"" << doc._date << "\" \"";
-    std::transform(begin(doc._name), end(doc._name), out, ::tolower);
+    std::transform(begin(doc._name), end(doc._name), out, std::tolower);
     stream << " " << doc._version << "\" \"" << doc._manTitle << "\"\n";
 
     // Print NAME section.
@@ -1319,7 +1319,7 @@ void ManToolDocPrinter_::print(std::ostream & stream, ToolDoc const & doc)
         {
             ToolDocSection_ const * sec = static_cast<ToolDocSection_ const *>(entry);
             stream << ".SH ";
-            std::transform(begin(sec->_title), end(sec->_title), out, ::toupper);
+            std::transform(begin(sec->_title), end(sec->_title), out, std::toupper);
             stream << "\n";
             isFirstInSection = true;
         }

